go-controller/pkg/config/config.go

```diff
@@ -152,8 +152,9 @@ var (
 			V4OVNServiceHairpinMasqueradeIP: net.ParseIP("169.254.169.5"),
 			V6OVNServiceHairpinMasqueradeIP: net.ParseIP("fd69::5"),
 		},
-		UserDefinedNetworkConntrackMarkBase: 2,
-		UserDefinedNetworkVRFTableBase:      2,
+		UserDefinedNetworkConntrackMarkBase:    2,
+		UserDefinedNetworkVRFTableBase:         2,
+		UserDefinedNetworkMasqueradeSubnetBase: 5,
```

Comment:
this is because .1,.2,.3 and .4 are used right?
I'd start from 10 actually so that .5 - .10 can still be used for other features if need be, aka placeholders

Reply:
.10 make sense to me.